ui/apps/platform/src/Containers/VulnMgmt/Reports/Components/ScopeName.tsx

```diff
@@ -3,14 +3,17 @@ import { Button, ButtonVariant, Spinner } from '@patternfly/react-core';
 
 import LinkShim from 'Components/PatternFly/LinkShim';
 import { getEntityPath } from 'Containers/AccessControl/accessControlPaths';
+import useFeatureFlags from 'hooks/useFeatureFlags';
 import useFetchScopes from 'hooks/useFetchScopes';
+import useCollection from 'Containers/Collections/hooks/useCollection';
 import { getAxiosErrorMessage } from 'utils/responseErrorUtils';
+import { collectionsBasePath } from 'routePaths';
 
 type ScopeNameProps = {
     scopeId: string;
 };
 
-function ScopeName({ scopeId }: ScopeNameProps): ReactElement {
+function ResourceScopeName({ scopeId }: ScopeNameProps): ReactElement {
```

Comment:
"Resource" was only a distinction used when we were abusing AccessScope by using it in VM reports.
It probably doesn't matter since this is going away after Collections goes live, but if we were going to rename ScopeName, `AccessScopeName` would be the parallel construction to `CollectionScopeName` below.